### PR TITLE
Add new release doc for 2.0.2

### PR DESCRIPTION
**Pull request by zhongjiajie**
Would change to formal PR until 2.0.2 release

TODO

* [x] Change to real release date in download.md until release vote pass
* [x] Need to wait until [Bug-7389][docs-zh][docs] Fix System User Manual link failure in version 2.0.1 and version 2.0.0 #609 merged
* [x] Pick dev change should merge to 202
* [x] Add how to start python gateway server
* [x] Auto upgrade database script https://github.com/apache/dolphinscheduler/pull/7530
* [x] Remove ambari context, Related PR https://github.com/apache/dolphinscheduler/pull/7749
* [x] Change mysql version https://github.com/apache/dolphinscheduler-website/pull/617

### CODE REVIEW

File: download/en-us/download.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ Use the links below to download the Apache DolphinScheduler from one of our mirr
 ## DolphinScheduler
 | Date | Version| | Downloads |
 |:---:|:--:|:--:|:--:|
+| Jan. 1st, 2022 | 2.0.2 | Source code| [[src]](https://www.apache.org/dyn/closer.lua/dolphinscheduler/2.0.2/apache-dolphinscheduler-2.0.2-src.tar.gz) [[asc]](https://downloads.apache.org/dolphinscheduler/2.0.2/apache-dolphinscheduler-2.0.2-src.tar.gz.asc) [[sha512]](https://downloads.apache.org/dolphinscheduler/2.0.2/apache-dolphinscheduler-2.0.2-src.tar.gz.sha512)|
+| | | Binary Distribution| [[bin]](https://www.apache.org/dyn/closer.lua/dolphinscheduler/2.0.2/apache-dolphinscheduler-2.0.2-bin.tar.gz) [[asc]](https://downloads.apache.org/dolphinscheduler/2.0.2/apache-dolphinscheduler-2.0.2-bin.tar.gz.asc) [[sha512]](https://downloads.apache.org/dolphinscheduler/2.0.2/apache-dolphinscheduler-2.0.2-bin.tar.gz.sha512)|
```
Comment (zhongjiajie) [inline download/en-us/download.md:11]: Change to real release date until release vote pass

**Comment by zhongjiajie** [comment]
* [ ] Need to wait until https://github.com/apache/dolphinscheduler-website/pull/609 merged

File: docs/zh-cn/2.0.2/user_doc/guide/installation/pseudo-cluster.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+mysql> GRANT ALL PRIVILEGES ON dolphinscheduler.* TO '{user}'@'localhost' IDENTIFIED BY '{password}';
+
+mysql> flush privileges;
+```
+
+完成上述步骤后，您已经为 DolphinScheduler 创建一个新数据库，现在你可以通过快速的 Shell 脚本来初始化数据库
+
+```shell
+sh script/create-dolphinscheduler.sh
+```
```
Comment (zhongjiajie) [inline docs/zh-cn/2.0.2/user_doc/guide/installation/pseudo-cluster.md:136]: I am quite sure about this, please may sure Shell script `script/create-dolphinscheduler.sh` work for version 2.0.2 @lenboo

**Review by dailidong — APPROVED** [review]
+1
good job

**Pull request merged**